netsuite/Module.md

```diff
@@ -0,0 +1,67 @@
+## Overview
+
+NetSuite's [SuiteTalk SOAP API](https://www.netsuite.com/portal/developers/resources/suitetalk-documentation.shtml) provides capability to access NetSuite operations related different kind of NetSuite records
```

Comment:
Suggestion: "NetSuite's [SuiteTalk SOAP API](https://www.netsuite.com/portal/developers/resources/suitetalk-documentation.shtml) provides the capability to access NetSuite operations related to different kinds of NetSuite records such as `Account`, `Client`, `Transactions`, `Invoice`, `Classifications` etc."